Library/Formula/znapzend.rb

```diff
@@ -0,0 +1,18 @@
+class Znapzend < Formula
+  desc "zfs backup with remote capabilities and mbuffer integration"
+  homepage "http://www.znapzend.org"
+  url "https://github.com/oetiker/znapzend/releases/download/v0.15.3/znapzend-0.15.3.tar.gz"
+  sha256 "27f7c218f4e71beac7b2ba52386a7742506d45c0b3fd40104f44ccef0f373b6e"
+
+  depends_on "perl" if MacOS.version <= :mavericks
+
+  def install
+    system "./configure", "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/znapzend", "-h"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!